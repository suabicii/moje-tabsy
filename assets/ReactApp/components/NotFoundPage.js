import React from "react";

function NotFoundPage() {
    return (
        <div id="notFound" className="alert alert-danger">
            <h1 className="text-center text-white">Nie znaleziono strony ðŸ˜¥</h1>
        </div>
    );
}

export default NotFoundPage;