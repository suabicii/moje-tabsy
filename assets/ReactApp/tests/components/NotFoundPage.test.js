import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import NotFoundPage from "../../components/NotFoundPage";

it('should correctly render NotFoundPage', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<NotFoundPage/>);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
});