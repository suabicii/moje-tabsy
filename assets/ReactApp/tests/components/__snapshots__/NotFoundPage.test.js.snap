// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render NotFoundPage 1`] = `
<div
  className="alert alert-danger"
  id="notFound"
>
  <h1
    className="text-center text-white"
  >
    Nie znaleziono strony ðŸ˜¥
  </h1>
</div>
`;
